### ISSUE TITLE
[Bug][a11y]: Slider quiet and indeterminate variants with editable rendering improperly

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3324]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Slider

### Expected behavior

Quiet and Indeterminate variants of Slider should render the track, and position the handle at 50% to match default value in the NumberField.

### Actual behavior

Quiet and Indeterminate variants of Slider fail to render the track and position the handle incorrectly when initialized with no value, min, or max as in the documentation examples at https://opensource.adobe.com/spectrum-web-components/components/slider/#quiet.

<img width="557" alt="Screenshot showing Slider Quiet and Indeterminate documentation examples" src="https://github.com/adobe/spectrum-web-components/assets/154077/dd8c2b79-83ae-4529-a2f7-71cb2f8d62e7">

Also, the focused state renders with a single blue border across the entire bottom of the component:

<img width="557" alt="Screenshot showing focused state of Slider Quiet variant" src="https://github.com/adobe/spectrum-web-components/assets/154077/017f888b-5450-4431-8e9c-b27b5f923b17">

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
